packages/core/admin/admin/src/translations/de.json

```diff
@@ -203,6 +322,8 @@
   "app.components.PluginCard.compatible": "Mit dieser App kompatibel",
   "app.components.PluginCard.compatibleCommunity": "Mit der Community kompatibel",
   "app.components.PluginCard.more-details": "Mehr Details",
+  "app.components.ToggleCheckbox.off-label": "Aus",
+  "app.components.ToggleCheckbox.on-label": "An",
```

Comment:
We recently changed the values to be `True` and `False`. Do you think "An" / "Aus" is the best translation? I'm wondering what others have, since "wahr" and "falsch" sounds weird to me.

Reply:
Tought about that as well... I think True/False or Wahr/Falsch are very general purpose, which is good in this case. However, they sound pretty "techy" (considering they are mostly used in user-facing components, not in say developer facing terminal output). Can only think of Ja/Nein (Yes/No) as another option, I don't know well that fits in.

Reply:
Thanks for elaborating. I'm leaning towards "Ja" and "Nein". We had "on/off" since the beginning of v4 and it confused many users in different contexts. "Ja/ Nein" seems a good compromise to me - what do you think?

Reply:
Yeah, I agree